source/quick-start.txt

```diff
@@ -2,5 +2,173 @@
 Quick Start
 ===========
 
-
 .. default-domain:: mongodb
+
+Check for Node and NPM
+----------------------
+
+Ensure your system has Node version 8 or later and NPM (Node Package
+Manager) installed. Run the following commands to check thatr you meet
+these requirements:
+
+First, check the version of Node
```

Comment:
Should end with some form of punctuation whether `:` or `.`

Reply:
Also a couple lines beneath this. Fixed both.